server/conn.go

```diff
@@ -1183,6 +1113,7 @@ func (cc *clientConn) handleLoadStats(ctx context.Context, loadStatsInfo *execut
 // There is a special query `load data` that does not return result, which is handled differently.
 // Query `load stats` does not return result either.
 func (cc *clientConn) handleQuery(ctx context.Context, sql string) (err error) {
+	ctx = context.WithValue(ctx, executor.LoadDataInput, cc)
```

Comment:
Could you use some other method to pass the argument?

Reply:
I tried to pass this as DataStream
interface during the BuildExecutor phase(or other execution method phase), but there seems to be recursive import problem , is there some better way to solve this ?